src/book/02-system/02-subsystems/06-behaviour.mdx

```diff
@@ -102,7 +102,7 @@ To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/bl
 
 ### Script Runner
 
-[ScriptRunner](https://github.com/NUbots/NUbots/tree/96357855b059cd499021552b3e25a4b158828dc5/module/behaviour/tools/ScriptRunner) is a purpose-level module. It takes the name of one or more script files as arguments and attempts to run the scripts. It does not take file paths, only the file name/s of the script/s to execute.
+[ScriptRunner](https://github.com/NUbots/NUbots/tree/8bfdde1ac3f51764515908dd514e653f6b38cf04/module/purpose/ScriptRunner) is a purpose-level module. It takes the name of one or more script files as arguments and attempts to run the scripts. It does not take file paths, only the file name/s of the script/s to execute.
```

Comment:
Is there a reason these links don't point to `main` ?
https://github.com/NUbots/NUbots/tree/main/module/purpose/ScriptRunner

Reply:
I try to have permalinks in NUbook so that links don't break